src/ts/dom.ts

```diff
@@ -562,6 +562,14 @@ export class DOM {
     }
   }
 
+  /**
+   * Removes an inline CSS property if it exists
+   * @param propertyName name of the property to remove
+   */
+  removeInlineCss(propertyName: string): void {
```

Comment:
nit: this could just be `removeCss` - the fact that it gets called on a component already implies that it is about inline css :)

Reply:
Rename: [Rename function to removeCss](https://github.com/bitmovin/bitmovin-player-ui/pull/538/commits/7b8b96eb9b08aea85acd17ced6010cf8f34ac633)